packages/core/src/components/private/BaseInputRange/styles.ts

```diff
@@ -0,0 +1,108 @@
+import { StyleSheet } from '../../../hooks/useStyles';
+import toRGBA from '../../../utils/toRGBA';
+
+export const HANDLE_SIZE = 14;
+export const HALF_HANDLE_SIZE = HANDLE_SIZE / 2;
+export const TRACK_HEIGHT = 4;
+export const MARK_SIZE = TRACK_HEIGHT;
+
+export const stylesheetInputRange: StyleSheet = ({ unit, ui, color, font, pattern }) => ({
+  container: {
+    position: 'relative',
+    height: 3 * unit,
+  },
+
+  container_disabled: {
+    opacity: 0.4,
+  },
+
+  input: {
+    transform: 'translateY(-2px)',
```

Comment:
I really couldn't figure out what determined the slider height in the browser, so this is empirical 😭